inspect/README.md

```diff
@@ -0,0 +1,195 @@
+# forta-inspector-bot
+
+Inspects run environment capabilities and limits
+
+Moved from [forta-network/forta-inspector-bot](forta-inspector-bot-repo)
+
+# Inspection Details
+
+## Scan API Inspection
+
+### `scan-api.accessible`
+
+Checks Scan APIs accessibility by establishing rpc connection and asking for current block height
+
+`1` for accessible, `-1` for not
+
+### `scan-api.module.eth`
+
+The scan node should either declare `eth` module in `rpc_modules` method or it should return a valid response to
+a `eth_chainId` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.module.net`
+
+The scan api should either declare `net` module in `rpc_modules` method or it should return a valid response to
+a `net_version` query
+
+`1` for supports, `-1` for not
+
+### `scan-api.block-by-number.hash`
+
+The hash of the block data retrieved from the scan API
+
+## Trace API Inspection
+
+### `trace-api.accessible`
+
+Checks trace APIs accessibility by establishing rpc connection and asking for current its chain id.
+
+`1` for accessible, `-1` for not
+
+### `trace-api.supported`
+
+Only necessary for Ethereum Mainnet.
+
+Returns `1` by default on other chains
+
+Checks trace APIs accessibility by establishing rpc connection and tracing the genesis block or the trace field in
+provided gRPC message
+
+`1` for accessible, `-1` for not
```

Comment:
```suggestion
`1` for supported, `-1` for not
```

Reply:
I feel like this part is a bit vague in general. In a typical inspection, what we should check is try to access the given trace api and check for trace support. 

However, there is a bot-usecase-specific workaround for that since it bots don't have direct access to trace api.

We may want to remove the part about "trace field in provided gRPC message"

Reply:
I just felt like we need to say "supported" here because we mention "accessible" in a separate result field.

> We may want to remove the part about "trace field in provided gRPC message"

Yeah, we could simplify it further. If I was doing something with the results, I would expect this to tell me something is supported or not, plain and simple. I wouldn't care too much about the encapsulated complexity i.e. how it was tested - it's probably better to make that visible in the code or code comments. What do you think about this change? https://github.com/forta-network/forta-core-go/pull/80#discussion_r929895370

Reply:
lgtm